packages/csvviewer-extension/src/index.ts

```diff
@@ -2,74 +2,112 @@
 // Distributed under the terms of the Modified BSD License.
 
 import {
-  ILayoutRestorer, JupyterLab, JupyterLabPlugin
+    ILayoutRestorer, JupyterLab, JupyterLabPlugin
```

Comment:
Can you get rid of these unnecessary whitespace changes? We use 2 spaces for indentation.

Reply:
Yes, I will use 2 spaces from now and fix this file.